sdl2-sonic-drivers/src/drivers/MIDParser.hpp

```diff
@@ -1,30 +1,27 @@
 #pragma once
 
-#include <audio/scummvm/Mixer.hpp>
 #include <audio/MIDI.hpp>
 #include <audio/midi/types.hpp>
 #include <memory>
 #include <cstdint>
 
+#include <RtMidi.h> // todo remove
```

Comment:
midparser can be removed ...

Reply:
but it could be used to split some stuff around midi... for now is no harm to keep it neither...